jdaviz/core/template_mixin.py

```diff
@@ -1490,12 +1491,14 @@ def _update_subset(self, subset, attribute=None):
                               else self._subset_to_dict(subset)
                               for s in self.items]
 
-        if attribute == 'subset_state' and subset.label == self.selected:
+        if attribute == 'subset_state' and subset.label in self.selected:
```

Comment:
I think we should be more explicit here (to avoid a substring match), maybe something like:

```suggestion
        if attribute == 'subset_state' and ((self.is_multiselect and subset.label in self.selected) or (subset.label == self.selected):
```

Reply:
Ah yes, this was leftover from the previous implementation. I'll use your suggestion instead.